packages/cashier-v2/src/constants/routesConfig.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable sort-keys */
+import { DummyRoute } from '../components';
+import { Cashier } from '../containers';
+
+export interface IRouteConfig {
+    component: React.ComponentType<IRouteConfig>;
+    path: string;
+    routes?: IRouteConfig[];
+    title: string;
+}
+
+export type TRouteComponent = React.ComponentProps<IRouteConfig['component']>;
+
+export const routesConfig: IRouteConfig[] = [
+    {
```

Comment:
I'm wondering, if there are reasons for extracgint routes config to file? 
Or can it just be configured within the .jsx to make it simpler?

Reply:
@wojciech-deriv 
moved configuration to the Router.tsx
create namespace for route types to store them in one place

Reply:
sorted